source/mobile/build-custom-mobile-apps.rst

```diff
@@ -0,0 +1,84 @@
+Building and Distributing Your Own Custom Mattermost Mobile Apps
+================================================================
+
+You can build and distribute custom versions of the Mattermost Mobile App. Choosing this approach means `you've decided not to use the mobile app Mattermost has made available <https://docs.mattermost.com/mobile/use-prebuilt-mobile-apps.html>`__ through public app stores.
+
+This approach is recommended for:
+
+- Organizations that want to customize their Mattermost Mobile Apps.
+- Customers using self-hosted Mattermost Team Edition or Enterprise Editions that prefer to host their own push proxy server instead of using one of Mattermost’s hosted versions.
+  
+Deployment Options
+------------------
+
+When you decide to build your own Mattermost Mobile apps, you have multiple ways to deploy it: 
+
+- Submitting your app to `an Enterprise App Store <#using-an-enterprise-app-store>`_.
+- Using `an Enterprise Mobile Management (EMM) provider <https://docs.mattermost.com/mobile/deploy-mobile-apps-using-emm-provider.html>`__.
+- Submitting your app to `public app stores <#using-public-app-stores>`_.
+- Using `another distribution method <#using-another-distribution-method>`_.
+
+Developing and Maintaining Your Custom App
+------------------------------------------
+
+The Mattermost Mobile App is an open source project. Customizing Mattermost Mobile apps requires a fork of the source code. Your team will be responsible for maintaining that fork, as well as keeping that fork updated with any changes made by Mattermost.
+
+This means that you manage the maintenance of your custom Mattermost Mobile apps, such as rebuilding and incorporating feature and/or security updates. If this isn't done regularly, your applications won't match the functionality of our publicly-available applications, and could be incompatible with future versions of Mattermost Server.
+
+This process can be complicated and can greatly increase deployment time, not only initially, but whenever the apps need to be updated. We recommend having your development team `review the Mattermost Mobile Apps developer documentation <https://developers.mattermost.com/contribute/mobile/>`__ to ensure they understand the scale and requirements of taking this path. This documentation provides guidance on building, compiling, signing, and customizing Mattermost Mobile apps.
```

Comment:
```suggestion
This process can be complicated and can greatly increase deployment time, not only initially, but whenever the apps need to be updated. We recommend having your development team `review the Mattermost Mobile Apps developer documentation <https://developers.mattermost.com/contribute/mobile/>`__ to ensure they understand the scale and requirements of taking this path. This documentation provides guidance on building, compiling, signing, and white labeling Mattermost Mobile apps.
```

Customizing is too big, it can mean adding code or functionality and that isn't covered by the dev docs